app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ActiveRecord::Base
+  validates :email, uniqueness: { case_sensitive: false }
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
```

Comment:
Also I think we might need ':database_authenticable' since that would be the way AV users not using Oauth would need to see their codes.